File: from-1901-to-2000/1957-delete-characters-to-make-fancy-string/solution.go
func makeFancyString(s string) string {
    r := []byte{}
    for i := 0; i < len(s); i ++ {
        if i > 1 && s[i] == s[i-1] && s[i] == s[i-2] {
            continue
        }
        r = append(r, s[i])
    }
    return string(r)
}

File: from-1901-to-2000/1957-delete-characters-to-make-fancy-string/solution.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        result = ""
        for i in range(len(s)):
            if i > 1 and s[i] == s[i - 1] and s[i] == s[i - 2]:
                continue
            result += s[i]

        return result